models/widgets.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose')
+const { Schema } = mongoose
+
+const WidgetSchema = new Schema ({
+  type: String,
+  title: String,
+  size:  {type: Number, required: true},
+  contents: String
```

Comment:
Also, I think we should name it something like `fields` - `contents` feels too generic...